engine/api/warning.go

```diff
@@ -0,0 +1,55 @@
+package api
+
+import (
+	"bytes"
+	"context"
+	"net/http"
+	"text/template"
+
+	"github.com/gorilla/mux"
+
+	"github.com/ovh/cds/engine/api/warning"
+	"github.com/ovh/cds/sdk"
+)
+
+func (api *API) getWarningsHandler() Handler {
```

Comment:
i miss the sdk part of it, and probably the warning display in cdsctl (can be done later)